package.json

```diff
@@ -11,46 +11,49 @@
     "test:unit": "npm run test:unit:_run -- --grep=$MOCHA_GREP",
     "nodemon": "nodemon --config nodemon.json",
     "lint": "node_modules/.bin/eslint .",
-    "format": "node_modules/.bin/prettier-eslint '**/*.js' --list-different",
-    "format:fix": "node_modules/.bin/prettier-eslint '**/*.js' --write"
+    "format": "node_modules/.bin/prettier-eslint $PWD'/**/*.js' --list-different",
+    "format:fix": "node_modules/.bin/prettier-eslint $PWD'/**/*.js' --write"
   },
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "async": "^0.1.22",
-    "coffee-script": "^1.7.1",
-    "express": "3.1.0",
-    "logger-sharelatex": "^1.7.0",
-    "metrics-sharelatex": "^2.2.0",
-    "mongojs": "^2.4.0",
-    "node-statsd": "0.0.3",
-    "request": "^2.65.0",
+    "async": "^2.6.3",
+    "body-parser": "^1.19.0",
+    "coffee-script": "^1.12.7",
+    "errorhandler": "^1.5.1",
+    "express": "4.17.1",
+    "logger-sharelatex": "^1.9.1",
+    "method-override": "^3.0.0",
+    "metrics-sharelatex": "^2.6.2",
+    "mongojs": "^3.1.0",
```

Comment:
This version bump requires a change. There is an error when adding a notification: `the update operation document must contain atomic operators`. 

Changing the 2nd parameter below to `{ $set: { ...doc } }` should fix this. Looks like all other `update` methods include an operator. 
https://github.com/overleaf/notifications/blob/2de8dbaf507bac8822f759a3f87429069a3af45f/app/js/Notifications.js#L88-L93

Reply:
Yeah, that error message is displayed on missing `$set` on updates (seen plenty of those during dependency upgrades). Good catch, thanks!!